mesonbuild/optinterpreter.py

```diff
@@ -23,7 +23,7 @@
 
 forbidden_option_names = set(coredata.builtin_options.keys())
 forbidden_prefixes = [lang + '_' for lang in compilers.all_languages] + ['b_', 'backend_']
-reserved_prefixes = ['cross_', 'build_']
+reserved_prefixes = ['cross_', 'build.']
```

Comment:
`build.` is not actually needed any more. Option names can not have dots in them (I tested this just now and it behaves correctly).

Reply:
oh haha! didn't mean to change that line. (I think reserving `build_`) is still a good idea.